packages/test-runner-module-mocking/tsconfig.json

```diff
@@ -33,6 +33,9 @@
   ],
   "include": [
     "src",
-    "test/**/*.ts"
   ],
+  "exclude": [
+    "browser",
+    "test"
+  ]
```

Comment:
why do you exclude these?

Reply:
This was just the same as the other projects. I removed it here, as it seems it can build without the exclude list.